src/Notice.tsx

```diff
@@ -127,15 +167,46 @@ export const Notice = memo(
         return (
             <div
                 id={id}
-                className={cx(fr.cx("fr-notice", `fr-notice--info`), classes.root, className)}
+                className={cx(
+                    fr.cx(
+                        "fr-notice",
+                        `fr-notice--${severity}`,
+                        doNotDisplayIcon && "fr-notice--no-icon"
+                    ),
+                    classes.root,
+                    className
+                )}
                 {...(refShouldSetRole.current && { "role": "notice" })}
                 ref={ref}
                 style={style}
                 {...rest}
             >
                 <div className="fr-container">
                     <div className="fr-notice__body">
-                        <p className={cx(fr.cx(`fr-notice__title`), classes.title)}>{title}</p>
+                        <p>
+                            <span className={cx(fr.cx(`fr-notice__title`), classes.title)}>
+                                {title}
+                            </span>
+                            {description && (
+                                <span className={cx(fr.cx("fr-notice__desc"), classes.description)}>
+                                    {description}
+                                </span>
+                            )}
+                            {link && (
+                                <Link
```

Comment:
For this link, using `title` is recommended to enhance a11y. Do you think we should add it as mandatory in Props ?